### DISCUSSION TITLE
How can I dowload swagger file hosted through oas-tools?

### DISCUSSION THREAD

**Discussion by uteja1** [oas-tools/oas-tools.github.io#26] [Q&A]
![image](https://user-images.githubusercontent.com/77608798/229712196-a83d75f1-2a6c-4b8d-a2fc-253ee98ebbcc.png)

I am unable to download the swagger file, from swagger UI

Can you please guide me, to make swagger file downloadable from swagger UI?

currently using version **"@oas-tools/core": "^3.1.0"**,

**Comment by alesancor1** [comment]
You could expose the swagger file to an endpoint using express:
```javascript
import fs from 'fs';

app.get('/docs/swagger.json', (req, res) => res.json(JSON.parse(fs.readFileSync('path/to/swagger.json'))))
```

**Reply by fwermus** [reply]
How can I expose it if it is a yaml file?

**Reply by uteja1** [reply]
const app = express();

app.get('/swagger-doc/openapi.yaml', (req, res) => {
      res.sendFile(path.join(__dirname, '/api/openapi.yaml'));
    });
    
    add this in app.js
